From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: v8-ci-autoroll-builder
 <v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Date: Tue, 17 Jan 2023 12:17:22 +0000
Subject: Update V8 to version 11.1.200.

Summary of changes available at:
https://chromium.googlesource.com/v8/v8/+log/cb5724ec..66d10810

Please follow these instructions for assigning/CC'ing issues:
https://v8.dev/docs/triage-issues

Please close rolling in case of a roll revert:
https://v8-roll.appspot.com/
This only works with a Google account.

CQ_INCLUDE_TRYBOTS=luci.chromium.try:linux-blink-rel
CQ_INCLUDE_TRYBOTS=luci.chromium.try:linux_chromium_chromeos_msan_rel_ng
CQ_INCLUDE_TRYBOTS=luci.chromium.try:linux_optional_gpu_tests_rel
CQ_INCLUDE_TRYBOTS=luci.chromium.try:mac_optional_gpu_tests_rel
CQ_INCLUDE_TRYBOTS=luci.chromium.try:win_optional_gpu_tests_rel
CQ_INCLUDE_TRYBOTS=luci.chromium.try:android_optional_gpu_tests_rel

R=hablich@chromium.org,vahl@chromium.org,v8-waterfall-sheriff@grotations.appspotmail.com

Change-Id: I2a7f8d5e082178cfeed599479144b5936f438390
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4172450
Bot-Commit: v8-ci-autoroll-builder <v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Commit-Queue: v8-ci-autoroll-builder <v8-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/heads/main@{#1093280}

diff --git a/DEPS b/DEPS
index 5c0b3c7e63dd6f57a7bf1accc43ac2b7ab84243a..6b148ca357b32ad69a92f97b7b9bcc83476ae6a7 100644
--- a/DEPS
+++ b/DEPS
@@ -308,7 +308,7 @@ vars = {
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling V8
   # and whatever else without interference from each other.
-  'v8_revision': 'cb5724ecc09929b9c248b035481130dc30ccc7a4',
+  'v8_revision': '66d10810dc149686026a90bae9ff1f8089031e32',
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling ANGLE
   # and whatever else without interference from each other.
